html/index.html

```diff
@@ -0,0 +1,480 @@
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&family=Urbanist:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+    </style>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+
+    <link rel="stylesheet" href="../css/index.css" />
+    <title>Document</title>
+
+    <style>
+        *{
+            font-family: Poppins;
+        }
+    </style>
+  </head>
+  <body>
+
+    <div class="wrapper">
+        <div class="top-container">
+
+
+            <div class="top-bar-connection">
+    
+                <div class="time-connection">
+    
+                    <img src="../assets/Bars/Status Bars/↳ Time.png" alt="">
+    
+                </div>
+    
+                <div class="b-w-b-connection">
+    
+                    <img src="../assets/Bars/Status Bars/Cellular Connection.png" alt="">
+                    <img src="../assets/Bars/Status Bars/Wifi.png" alt="">
+                    <img src="../assets/Bars/Status Bars/Battery.png" alt="">
+    
+                </div>
+    
+            </div>
+    
+            <div class="header">
+    
+    
+            
+    
+                <div class="left-text">My Logo</div>
+    
+                <div class="right-hamburger">
+    
+                    <img src="../assets/Union.png" alt="">
+    
+                </div>
+    
+            </div>
+    
+            <div class="top-body-background">
+                <div class="top-body">
+    
+                    <h2>I'm Toyyib, Glad to see you!</h2>
+            
+                    <p>
+                    I’m a software developer! I can help you build a product , feature or
+                    website Look through some of my work and experience! If you like what
+                    you see and have a project you need coded, don’t hestiate to contact
+                    me.
+                    </p>
+            
+                </div>
+            
+                <div class="footer-connect">
+                    
+                    <p><a href="#">LET'S CONNECT</a> </p>
+            
+                </div>
+            
+                <div class="top-footer">
+            
+                    <img src="../assets/twitter.svg" alt="">
+                    <img src="../assets/linkdn.svg" alt="">
+                    <img src="../assets/medium.svg" alt="">
+                    <img src="../assets/github.svg" alt="">
+                    <img src="../assets/rabbit.svg" alt="">
+            
+                </div>
+    
+            </div>
+    
+        </div>
+    
+        
+        <div class="second-container">
```

Comment:
- [ ] Nice work @Simpleshaikh1, however, there are a few things that you need to fix before you go ahead with your project, as stated in the requirement, the project's container must be a section tag for semantic reasons, while you have used divs, I have 2 propositions about this, 
- You can change the div on line 96 into the `section` tag, insert all projects in that single container and use the display: grid CSS method to display its contents(projects) 
- or you can wrap all the project div(second_container) in a section tag you will create and use a display grid on it as well. then make the necessary changes to update the design because currently, we can not even check it fine.

![5](https://user-images.githubusercontent.com/77843050/227930279-0ef0e6f0-9b4b-462d-b961-b781d8e441dd.gif)